internal/common/metrics.go

```diff
@@ -211,3 +212,34 @@ func SendEMFMetrics(metricPerInterval int, metricLogGroup, metricNamespace strin
 		}
 	}
 }
+
+// isAllValuesGreaterThanOrEqualToExpectedValue will compare if the given array is larger than 0
+// and check if the average value for the array is not la
+// TODO: Moving metric_value_benchmark to validator
+// https://github.com/aws/amazon-cloudwatch-agent-test/pull/162
+func IsAllValuesGreaterThanOrEqualToExpectedValue(metricName string, values []float64, expectedValue float64) bool {
```

Comment:
This exists in https://github.com/aws/amazon-cloudwatch-agent-test/blob/8189a84e2d3831d402650639f0ca74d36cb60048/test/metric/metric_validation_util.go#L8 which got merged in first

Reply:
Merged from main in the newest commit to use this method from `metric` package